.github/config.yml

```
@@ -0,0 +1,30 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
```

```suggestion
  Hello there!👋 Welcome to the Pixelvibe project!🚀⚡
```
A particular name looks much nice and presentable.